app/assets/javascripts/discourse/app/components/sidebar/api_panels.hbs

```diff
@@ -0,0 +1,47 @@
+<div class="sidebar-sections">
+  {{#each this.customSections as |customSection|}}
+    <Sidebar::Section
+      @sectionName={{customSection.name}}
+      @headerLinkText={{customSection.text}}
+      @headerLinkTitle={{customSection.title}}
+      @headerActionsIcon={{customSection.actionsIcon}}
+      @headerActions={{customSection.actions}}
+      @willDestroy={{customSection.willDestroy}}
+      @collapsable={{@collapsableSections}}
+      @displaySection={{customSection.displaySection}}
+    >
+
+      {{#each customSection.links as |link|}}
+        <Sidebar::SectionLink
+          @linkName={{link.name}}
+          @class={{link.classNames}}
+          @route={{link.route}}
+          @model={{link.model}}
+          @models={{link.models}}
+          @title={{link.title}}
+          @contentCSSClass={{link.contentCSSClass}}
+          @prefixColor={{link.prefixColor}}
+          @prefixBadge={{link.prefixBadge}}
+          @prefixType={{link.prefixType}}
+          @prefixValue={{link.prefixValue}}
+          @prefixCSSClass={{link.prefixCSSClass}}
+          @suffixType={{link.suffixType}}
+          @suffixValue={{link.suffixValue}}
+          @suffixCSSClass={{link.suffixCSSClass}}
+          @hoverType={{link.hoverType}}
+          @hoverValue={{link.hoverValue}}
+          @hoverAction={{link.hoverAction}}
+          @hoverTitle={{link.hoverTitle}}
+          @currentWhen={{link.currentWhen}}
+          @didInsert={{link.didInsert}}
+          @willDestroy={{link.willDestroy}}
+          @content={{link.text}}
+          @contentComponent={{component
+            link.contentComponent
+            status=link.contentComponentArgs
+          }}
+        />
+      {{/each}}
```

Comment:
I think we're duplicating this with what is displayed in custom sections. It'll be good if we can DRY up the code here.